src/sentry/utils/pytest/relay.py

```diff
@@ -52,6 +52,7 @@ def relay_server_setup(live_server, tmpdir_factory):
         datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S_%f")
     )
     config_path = tmpdir_factory.mktemp(prefix)
+    config_path.chmod(0o755)
```

Comment:
pytest temp directories changed to be 0o700 -- this made them unreadable in the mounted container by the unprivileged user